src/pages/Gift/Purchase/Submit/index.tsx

```diff
@@ -10,19 +10,10 @@ import { SubmitStep, setStep } from "slices/gift";
 import { humanize } from "helpers";
 import { appRoutes } from "constants/routes";
 import CompleteBtn from "./CompleteBtn";
-import { estimateTx } from "./estimateTx";
 
 export default function Submit(props: WithWallet<SubmitStep>) {
   const dispatch = useSetter();
-  const [estimate, setEstimate] = useState<EstimateStatus>("loading");
```

Comment:
not use if giftcards still has web3 nature so just left most of the existing UI and web3 wiring

Reply:
@ap-justin Gift Cards will be 100% AWS driven now. @jp-mariano has removed the `deposit_id` field already in AWS that the contracts used. Now, a giftcard balance will simply need to be reduced as the user spends it. This'll need some re-thinking of the UI and some changes in AWS to make sure we are correctly tracking remaining spendable balances (ie. $100 USD initially finalized and later claimed by a user with the code could be spend in multiple chunks like two donations "with a gift card" of $25 & $75 each). 

I'd suspect we end up adding GiftCards as an Payment Option on donations, under an "Other" category, coming after existing "Credit Card" and "Crypto" tabs. An "Other" tab is where we'll add "DAF" and "Stocks" for those donations options (eventually).